src/chickens_everest.c

```diff
@@ -36,6 +36,50 @@ static void init_common_everest(void)
     reg_clr(s3_0_c15_c2_4, BIT(0) | BIT(1) | BIT(16) | BIT(17) | BIT(18) | BIT(22));
 }
 
+void init_t6030_everest(int rev)
+{
+    UNUSED(rev);
+    msr(s3_1_c15_c1_5, 0x1uL);
```

Comment:
this uses a different value than `init_t6030_sawtooth()`. I think this is the cause for your believe that performance and efficiency cores behave differently.
I think we should do something like https://github.com/jannau/m1n1/commit/80381504ed905dadc4eaa5f8369410b0aad73b1a to avoid this issue

Reply:
I will leave this out for now and will rebase once your changes are upstream